packages/rpc-provider/src/ws/Provider.ts

```diff
@@ -65,7 +65,7 @@ const l = logger('api-ws');
 export default class WsProvider implements WSProviderInterface {
   readonly #coder: Coder;
 
-  readonly #endpoint: string;
+  readonly #endpoint: string[];
```

Comment:
```suggestion
  readonly #endpoints: string[];
```

Since it is now always multiple.